packages/server/src/service/mg.ts

```diff
@@ -92,6 +94,50 @@ export const sendSignupEmail = async (
   return mg.messages().send(emailData)
 }
 
+
+export const sendCSVEmail = async (
+  csv: string,
+  uid: string,
+  name: string | undefined,
+  emails: string[],
+  oid: string,
+) => {
+  try {
+    const attachment = [new mg.Attachment({
+      data: Buffer.from(csv),
+      contentType: 'text/csv',
+      filename: 'saved_sign_ups.csv',
+    })]
+    const mgData = {
+      from: processEnvOrThrow('MG_FROM_ADDR'),
+      'h:Reply-To': [processEnvOrThrow('MG_REPLY_TO_ADDR'), ...emails].join(','),
+      'v:metadata': { oid },
+    }
+    const email: mailgun.messages.SendData = {
+      to: emails,
+      subject: `VoteByMail.io ${oid} saved registrations`,
+      text:
+        `Hello ${name ?? 'Organizer'},
+
+        As per request on VoteByMail.io organizers dashboard, we've sent ${oid} registrations to your email address, please check the attachments of this email.`.replace(/( ){2,}/, ''),
```

Comment:
Added this regexp to remove the extra white-spacing from indenting.

Reply:
thanks! sorry didn't realize it would affect the actual output